sycophancy_analysis/cli/app.py

```
@@ -0,0 +1,263 @@
+    temperature: float = typer.Option(0.2, help="Base sampling temperature."),
+    system_prompt: Optional[str] = typer.Option(None, "--system-prompt", "--system_prompt", help="Optional system prompt."),
+    knn_k: int = typer.Option(8, "--knn-k", "--knn_k", help="k for kNN graph."),
+    leiden_resolution: float = typer.Option(1.0, "--leiden-resolution", "--leiden_resolution", help="Leiden resolution for clustering."),
+    bridge_threshold: float = typer.Option(0.5, "--bridge-threshold", "--bridge_threshold", help="Bridge edge threshold when plotting network."),
+    save_prefix: Optional[str] = typer.Option(None, "--save-prefix", "--save_prefix", help="Prefix for saving outputs."),
+    interactive: bool = typer.Option(False, "--interactive", help="Launch scoring configuration wizard."),
+    include_slugs: Optional[str] = typer.Option(None, "--include-slugs", "--include_slugs", help="Comma-separated slugs to include."),
+    exclude_slugs: Optional[str] = typer.Option(None, "--exclude-slugs", "--exclude_slugs", help="Comma-separated slugs to exclude."),
+    include_names: Optional[str] = typer.Option(None, "--include-names", "--include_names", help="Comma-separated model names to include."),
+    exclude_names: Optional[str] = typer.Option(None, "--exclude-names", "--exclude_names", help="Comma-separated model names to exclude."),
+    export_prompts: Optional[Path] = typer.Option(None, "--export-prompts", "--export_prompts", help="Write prompt battery JSON then exit."),
```

Ruff (0.13.1)

Do not perform function call `typer.Option` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)